cd/cd.yaml

```diff
@@ -528,14 +528,14 @@ Resources:
               RunOrder: 1
         - Name: Staging
           Actions:
-            - Name: StagingDeploy
+            - Name: CreateStagingChangeSet
               ActionTypeId:
                 Category: Deploy
                 Owner: AWS
                 Provider: CloudFormation
                 Version: 1
               Configuration:
-                ActionMode: REPLACE_ON_FAILURE
+                ActionMode: CHANGE_SET_REPLACE
```

Comment:
Is this an unrelated change?

Reply:
Or is this just to separate creating/executing the changeset, so you can notify in between?

Reply:
oh hm

Reply:
that probably should change back. replace is just a more aggressive method, so if staging were to get stuck in a bad way another deploy could just replace it, rather than trying to update which always needs to happen from a stable state